public/app/pipeline-orchestrator-worklet-init.js

```
@@ -0,0 +1,294 @@
+        if (!el) continue;
+
+        const isRT = el.classList.contains('realtime');
+
+        el.addEventListener('input', () => {
+          const v = parseFloat(el.value);
+          if (isRT) {
+            // Real-time: post immediately
+            this._postParam(workletKey, v);
+          } else {
+            // Non-RT: debounce to 16ms
+            clearTimeout(this._debTimers[sliderId]);
```

ESLint

[error] 'clearTimeout' is not defined.

(no-undef)